### PR TITLE
cra pwa and initial assets

### PR DESCRIPTION
**Pull request by saulyz**
CRA with typescript and progressive web app features
favicons, logos and manifest file update

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+.eslintcache
```
Comment (rikius) [inline .gitignore:25]: add new line to the end of file

File: package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "pets-front",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.6",
```
Comment (rikius) [inline package.json:6]: please move test dependencies to devDependency section:
- "devDependencies": Packages that are only needed for local development and testing.

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (rikius) [inline src/index.css:1]: we should need to add reset style. e.g.:

```

/* http://meyerweb.com/eric/tools/css/reset/ 
   v2.0 | 20110126
   License: none (public domain)
*/

html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed, 
figure, figcaption, footer, header, hgroup, 
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
	margin: 0;
	padding: 0;
	border: 0;
	font-size: 100%;
	font: inherit;
	vertical-align: baseline;
}
/* HTML5 display-role reset for older browsers */
article, aside, details, figcaption, figure, 
footer, header, hgroup, menu, nav, section {
	display: block;
}
body {
	line-height: 1;
}
ol, ul {
	list-style: none;
}
blockquote, q {
	quotes: none;
}
blockquote:before, blockquote:after,
q:before, q:after {
	content: '';
	content: none;
}
table {
	border-collapse: collapse;
	border-spacing: 0;
}
```

just need to decide what kind of reset we will use.

**Review by rikius — CHANGES_REQUESTED** [review]
Please update according comments in the PR.

**Review by rikius — APPROVED** [review]

**Comment by rikius** [comment]
some tasks will be moved with other project issues. so we are moving forward

File: package.json (hunk 2 above)
Comment (saulyz) [reply package.json:6]: let's ignore this, referring to this practice - https://stackoverflow.com/questions/44868453/create-react-app-install-devdepencies-in-dependencies-section

File: src/index.css (hunk 3 above)
Comment (saulyz) [reply src/index.css:1]: let's use https://material-ui.com/components/css-baseline/ and skip it in this task

**Review by rikius — APPROVED** [review]

**Review by rikius — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (saulyz) [reply .gitignore]: done

**Pull request merged**